grails-app/services/au/org/ala/bie/ImportService.groovy

```diff
@@ -567,18 +573,20 @@ class ImportService implements GrailsConfigurationAware {
             }
         }
         log("Committing to ${buffer.size()} documents to SOLR...")
-        indexService.indexBatch(buffer)
+        if (online) {
+            log "Search for wordpress pages may be temporarily unavailable"
+        }
+        indexService.deleteFromIndex(IndexDocType.WORDPRESS, online)
+        indexService.indexBatch(buffer, online)
         updateProgressBar(100, 100) // complete progress bar
         log "Finished wordpress import"
     }
 
     /**
      * Index Knowledge Base pages.
      */
-    def importKnowledgeBasePages() throws Exception {
-        log "Starting knowledge base import"
-        // clear the existing WP index
-        indexService.deleteFromIndex(IndexDocType.KNOWLEDGEBASE)
+    def importKnowledgeBasePages(boolean online) throws Exception {
```

Comment:
Doe it need the `boolean online = false` here like you've got below?

Reply:
IndexService parameters have been left without defaults, to allow the controller to explicitly set it.